ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -262,8 +262,34 @@ objects:
         elif [[ "${CLUSTER_TYPE}" == "azure4" ]]; then
           export TEST_PROVIDER='azure'
         elif [[ "${CLUSTER_TYPE}" == "openstack" ]]; then
+
+          # Can we just export OS_CONFIG_FILE and OS_CLOUD?
```

Comment:
Yes!  Export OS_CLOUD=$CLOUD and then make sure the clouds.yaml is in the working directory, or ~/.config/openstack/